File: Python/PRAK604-2210817120009-Najah_Maisyaroh.py
kata1=input()
kata2=input()
c=0
d=0
if(len(kata1)!=len(kata2)):
    print("Panjang kalimat berbeda, pesan palsu")
else:
    for i in range(len(kata1)):
        if(kata1[i]==kata2[i]):
            print("*", end="")
            c+=1
        else:
            print("#", end="")
            d+=1
    print()
    print("* = ", c)
    print("# = ", d)
    if(c>=d):
        print("Pesan Asli")
    else:
        print("Pesan Palsu")

File: C/PRAK604-2210817120009-Najah_Maisyaroh.c
#include <stdio.h>
#include <string.h>
int main(){
    char kata1[50], kata2[50];
    scanf(" %[^\n]%*c", &kata1);
    scanf(" %[^\n]%*c", &kata2);
    int len1, len2, i, c=0, d=0;
    len1 = strlen(kata1);
    len2 = strlen(kata2);
    if(len1!=len2){
        printf("Panjang kalimat berbeda, pesan palsu");}
    else{
        for(i=0;i<len1;i++){
            if(kata1[i]==kata2[i]){
                printf("*");
                c++;}
            else{
                printf("#");
                d++;}}
        printf("\n* = %d", c);
        printf("\n# = %d", d);
        if(c>=d){
            printf("\nPesan Asli");}
        else{
            printf("\nPesan Palsu");}}
}